src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+* {
+  padding: 0;
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+
+  --hover-color: #00acdc;
```

Comment:
Use a more descriptive name for this variable and create a variable with root selector